scripts/table_updates/update_data_table.py

```diff
@@ -399,13 +449,27 @@ def main():
     parser.add_argument(
         "-p", "--project_config", default="config.json", help="Project config file"
     )
+    parser.add_argument(
```

Comment:
Do we need to update `modules/update_data_table.nf`with these new args? If so, we probably also have to do a nextflow run of this step on staging project to test that that's working.

Reply:
Oh you're right. I forgot to push the change.